cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/NewCommand.java

```diff
@@ -110,12 +110,12 @@ public void execute() {
             Files.createDirectories(path);
             CommandUtil.initProject(path);
         } catch (AccessDeniedException e) {
-            errStream.println("error: Error occurred while creating project : " + "Insufficient Permission");
+            errStream.println("error: error occurred while creating the project: " + "insufficient permissions.");
```

Comment:
Not related to you PR, but we can change this to
```suggestion
            errStream.println("error: error occurred while creating the project: insufficient permissions.");
```

Reply:
Fixed.